sandbox/package.json

```diff
@@ -25,9 +25,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
This was causing problems with the react-app for some reason. Will revisit linting in a separate PR.